platform/security/secure-boot/src/test/groovy/org/codice/ddf/security/boot/SecureBootSpec.groovy

```diff
@@ -245,68 +245,4 @@ class SecureBootSpec  extends Specification {
         1 * systemService.halt("0")
         exitedDueToSystemServiceFailureToExit == false
     }
-
-    /**
-     * ie. /opt/ddfHomeSymLink -> /users/tomPenny/ddf
-     */
-    def 'System exits normally when security manager enabled and symbolic link outside of user home dir pointing to ddf installation inside of user home dir'() {
-        setup:
-        def exitedDueToSystemServiceFailureToExit = false
-        def optDir = temporaryFolder.newFolder("opt").toPath()
-        def tomPennyHomeDir = temporaryFolder.newFolder("tomPenny")
-        def ddfHomeDir = Files.createDirectory(Paths.get(tomPennyHomeDir.toString(), "ddf"))
-        def ddfHomeSymLink = Files.createSymbolicLink(Paths.get(optDir.toString(), "ddfHomeSymLink"), ddfHomeDir)
-        def systemService = Mock(SystemService)
-        def securityManager = Mock(SecurityManager)
-        System.properties.'ddf.home' = ddfHomeSymLink.toString()
-        System.properties.'user.home' = tomPennyHomeDir.toString()
-
-        when:
-        def secureBoot = new SecureBoot(systemService) {
```

Comment:
✏️ This could actually be done using a spy and then stubbing the particular methods like this:

```def secureBoot = Spy(SecureBoot, constructorArgs: [systemService])

1 * secureBoot >> securityManager```

Reply:
changed